toolchain/testing/testdata/min_prelude/bool.carbon

```diff
@@ -3,13 +3,15 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 // EXTRA-ARGS: --custom-core --exclude-dump-file-prefix=min_prelude/
+// TODO: This should move to the place where it's imported, but it ends up
+// included more than once and that fails to build.
```

Comment:
```suggestion
```

In this case, both are directly `export import`ed, so we'd want to keep the includes here